packages/presentation/lib/navigation/base_page.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/navigation/base_arguments.dart';
+
+class BasePage<T extends BaseArguments> extends Page {
+  const BasePage({
+    @required required LocalKey key,
```

Comment:
As of Dart 2.12, the required keyword replaces the @required, so I think this annotation is no need here

Reply:
removed